### PR TITLE
add TSVREFE bit on add and setchannel

### PR DESCRIPTION
**Pull request by victorandrehc**

### CODE REVIEW

File: src/modm/platform/adc/stm32/adc_impl.hpp.in [hunk 1]
```diff
@@ -97,6 +97,12 @@ modm::platform::Adc{{ id }}::setChannel(const Channel channel,
 	ADC{{ per }}->SQR3 = uint32_t(channel) & 0x1f;
 
 	setSampleTime(channel, sampleTime);
+%% if temperature_available
```
Comment (klsc-zeat) [inline src/modm/platform/adc/stm32/adc_impl.hpp.in:100]: maybe add the thing with the temperature and internal ref to the commit message :)

Comment (victorandrehc) [reply src/modm/platform/adc/stm32/adc_impl.hpp.in:100]: do you mean rephrase the commit message to something like "activate internal temperature and internal reference if the channel is added"?

Comment (klsc-zeat) [reply src/modm/platform/adc/stm32/adc_impl.hpp.in:100]: yep, exactly! or `add TSVREFE bit if temperature or internal reference channels is added`

Comment (victorandrehc) [reply src/modm/platform/adc/stm32/adc_impl.hpp.in:100]: Just did it

**Pull request merged**